### PR TITLE
ObjectMapper consistency

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
use JsonUtil.DEFAULT_MAPPER for consistency across the code base.

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: query/src/org/labkey/query/MultiValueTest.jsp [hunk 1]
```diff
@@ -98,7 +99,7 @@
         MockHttpServletResponse resp = ViewServlet.GET(url, getUser(), null);
         String content = resp.getContentAsString();
 
-        ObjectMapper om = new ObjectMapper();
+        ObjectMapper om = JsonUtil.DEFAULT_MAPPER.copy();
```
Comment (labkey-adam) [inline query/src/org/labkey/query/MultiValueTest.jsp]: Is `copy()` necessary?

File: query/src/org/labkey/query/MultiValueTest.jsp [hunk 2]
```diff
@@ -213,7 +214,7 @@
         MockHttpServletResponse resp = ViewServlet.GET(selectUrl, getUser(), null);
         String content = resp.getContentAsString();
 
-        ObjectMapper om = new ObjectMapper();
+        ObjectMapper om = JsonUtil.DEFAULT_MAPPER.copy();
```
Comment (labkey-adam) [inline query/src/org/labkey/query/MultiValueTest.jsp]: Is `copy()` necessary?

File: query/src/org/labkey/query/MultiValueTest.jsp [hunk 3]
```diff
@@ -304,7 +305,7 @@
         String content = resp.getContentAsString();
         //System.out.println("query response:\n" + content);
 
-        ObjectMapper om = new ObjectMapper();
+        ObjectMapper om = new ObjectMapper().copy();
```
Comment (labkey-adam) [inline query/src/org/labkey/query/MultiValueTest.jsp]: `JsonUtil.DEFAULT_MAPPER`? Why `copy()`?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Adjust MultiValueTest.jsp?

File: query/src/org/labkey/query/MultiValueTest.jsp (hunk 1 above)
Comment (labkey-matthewb) [reply query/src/org/labkey/query/MultiValueTest.jsp]: Coz that way I don't have to read the test to see if it mutates the ObjectMapper.

File: query/src/org/labkey/query/MultiValueTest.jsp (hunk 3 above)
Comment (labkey-matthewb) [reply query/src/org/labkey/query/MultiValueTest.jsp]: oops

**Review by labkey-adam — APPROVED** [review]

**Pull request merged**